platform/broadcom/sonic-platform-modules-mitac/ly1200-32x/opt/xcvr-serv/bms_xcvr_access.c

```diff
@@ -0,0 +1,345 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
the function here is overalpping with the sff_8436_eeprom driver in sonic.

Reply:
There are two main benefits to access EEPROM via the API of bms_xcvr_access.c
1) it is a portable implementation that does not depend on any EEPROM driver.
2) the execution performance is faster than calling three APIs open(), read()/write(), and close() to access sysfs.